fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoredRecord.java

```diff
@@ -54,25 +53,14 @@
     private final boolean split;
     private final boolean versionedInline;
 
-    // TODO: Remove these at some point once experiments are complete.
-    private final Optional<Long> timeToLoad;
-    private final Optional<Long> timeToDeserialize;
-
-    @SuppressWarnings("squid:S00107") // Allow this many args since mostly initialized by builder.
-    protected FDBStoredRecord(@Nonnull Tuple primaryKey, @Nonnull RecordType recordType, @Nonnull M record,
-                              @Nonnull int keyCount, int keySize, int valueSize, boolean split, boolean versionedInline, @Nullable FDBRecordVersion recordVersion) {
-        this(primaryKey, recordType, record, keyCount, keySize, valueSize, split, versionedInline, recordVersion, Optional.empty(), Optional.empty());
-    }
-
     public FDBStoredRecord(@Nonnull Tuple primaryKey, @Nonnull RecordType recordType, @Nonnull M record,
                            @Nonnull FDBStoredSizes size, @Nullable FDBRecordVersion recordVersion) {
         this(primaryKey, recordType, record, size.getKeyCount(), size.getKeySize(), size.getValueSize(), size.isSplit(), size.isVersionedInline(), recordVersion);
     }
 
     @SuppressWarnings("squid:S00107")
     public FDBStoredRecord(@Nonnull Tuple primaryKey, @Nonnull RecordType recordType, @Nonnull M record,
-                           @Nonnull int keyCount, int keySize, int valueSize, boolean split, boolean versionedInline, @Nullable FDBRecordVersion recordVersion,
```

Comment:
This constructor should probably be marked as `INTERNAL`.

Reply:
Done.